projects/igniteui-angular/src/lib/drop-down/drop-down.component.ts

```diff
@@ -561,6 +561,19 @@ export class IgxDropDownComponent extends IgxDropDownBaseDirective implements ID
         }
     }
 
+    /**
+     * Clears the selection of the dropdown
+     * ```typescript
+     * this.dropdown.clearSelection();
+     * ```
+     */
+    public clearSelection() {
+        if (this.selectedItem) {
+            this.selectedItem.selected = false;
```

Comment:
We should fire here `onSelection` event. The new item in the event args should be null and the old item should be currently selected item.
We should also check if event is canceled - if so we should not clear the selection.